src/FrameRipper.py

```diff
@@ -0,0 +1,79 @@
+import cv2
+import numpy as np
+import os 
+
+#Frame Ripper Script WIP
+#Working on adding subset functionality in order to randomly grab sets of frames instead of all of them
+#
+#Steve
+
+#Function to scan the video files, processes and adds them to Array video_list
+def getVideoFiles():
+    
+    entries = os.listdir('Videos')
+    video_list = []
+    
+    for entry in entries:
+        video_list.append("Videos/" + entry)
```

Comment:
This is of the *smallest* importance, but inconsistencies in quotes.
```suggestion
        video_list.append('Videos/' + entry)
```